### DISCUSSION TITLE
upload csv file along with yml script on aws lambda

### DISCUSSION THREAD

**Discussion by nikhil-git** [artilleryio/artillery#2098] [Artillery]
What is the command to run artillery script over aws lambda wherein the script takes a csv file as a payload. I am using below command to run the artillery script over aws lambda. 
artillery run-lambda \        
   --region ap-south-1 \
   --count 50 \
   nifty_100_202308281514.csv
   mf_fundamentals.yml

Here the script refers to a csv file but I am not sure how to bundle the csv test file data in this command. Can someone pls help with it?

**Comment by hassy** [comment]
hi @nikhil-git :wave: Use the `config.payload` section as normal ([docs](https://www.artillery.io/docs/reference/test-script#payload---loading-data-from-csv-files)), Artillery will take care of it automatically.

**Reply by nikhil-git** [reply]
Thanks @hassy I am aware about the config.payload section. However, I am not sure that when running the script over aws lambda, will artillery be bundling the csv file along with the original test script and sending it to the aws cloud.

**Reply by hassy** [reply]
@nikhil-git yes it will bundle it automatically and make it available to all Artillery workers running on AWS Lambda
